### PR TITLE
Update List to keep up with "Web Hosting" on LEC

### PR DESCRIPTION
**Pull request by stevenzhu25**
Update this list to keep up with latest version (150) on https://community.letsencrypt.org/t/web-hosting-who-support-lets-encrypt/6920/1

### CODE REVIEW

File: _data/hosting_providers.json [hunk 1]
```diff
@@ -1119,6 +1119,16 @@
     "reviewed": "",
     "note": ""
   },
+  {
+    "name": "Mijn",
+    "link": "https://mijn.host/",
+    "category": "partial",
+    "tutorial": "",
```
Comment (ohemorange) [inline _data/hosting_providers.json]: Is there a tutorial we could put here? We'd like to have tutorials linked to for `partial` entries.

Comment (jeffreylees) [inline _data/hosting_providers.json]:
```suggestion
    "tutorial": "https://mijn.host/kb/lets-encrypt-ssl-certificaat-aanvragen/",
```

Comment (ohemorange) [reply _data/hosting_providers.json]: Perfect, thanks for finding that!

**Comment by stevenzhu25** [comment]
Hi @ohemorange,

Sorry about the delay.
I've updated the list to the most recent version on LEC and added tutorial link found by @jeffreylees

**Pull request merged**